app/models/concerns/smooch_search.rb

```diff
@@ -213,7 +213,7 @@ def search_by_keywords_for_similar_published_fact_checks(words, after, team_ids,
     def send_search_results_to_user(uid, results, team_id)
       team = Team.find(team_id)
       redis = Redis.new(REDIS_CONFIG)
-      language = self.cached_user_language(uid)
+      language = self.get_user_language(uid, {}, nil, team_id)
```

Comment:
As noted above, maybe we don't need to pass a `team_id` here.

Reply:
Done